sql.go

```diff
@@ -0,0 +1,168 @@
+package main
```

Comment:
Do we still need this file?

Reply:
**Fixed.** Nope, that came from the merge zzz